boat-maven-plugin/README.md

```diff
@@ -1,9 +1,49 @@
 #boat maven plugin.
 
-Three mojos:
-- Generate
-- Export
-- compare
+The boat plugin has 9 goals:
+
+- bundle
```

Comment:
This is not implemented yet. I currently do not have an implemementation that is as good as the npm open api tools. I might have another go at it.

Reply:
Do you think it will be implemented by the time master is updated? I could either add a note about the non-implementation or remove.